tests/expected_outputs/public/efficientdet/retinanet_effd0_bifpn_1x_coco-10.json

```diff
@@ -0,0 +1 @@
+{"map": [0.404, 0.416]}
```

Comment:
What are these numbers? The second one in particular.

Reply:
Those were test numbers I forgot to update. Changed the values to align with the expected precision

Reply:
I guess 0.404 was correct...